Refresh pinnings

jakirkham opened the pull request:
Some of the pinnings here were out-of-date. This just refreshes them.

jakirkham committed:
MNT: Re-rendered with conda-smithy 2.4.5

jakirkham committed:
Refresh pins

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

jakirkham commented on recipe/meta.yaml:
```diff
@@ -13,29 +13,29 @@ source:
     - libxml2 2.9.*
-    - krb5
+    - krb5 1.14.*
     - curl
```
Noticed `curl` was in `run`, but not `build`. Should it be in `build`?

jakirkham committed:
Use libprotobuf

Instead of using `protobuf`, which contains C++ and Python support, just
use `libprotobuf`, which is C++ only. As this recipe doesn't seem to use
`python` anyways, this seems like a better choice.

jakirkham committed:
Bump build number to 1

Now that pins have been refreshed, bump the build number to 1.

martindurant commented:
@jakirkham , can you check if #16 also works?

jakirkham commented:
Not ATM. Am in the midst of updating several feedstocks in preparation for cb3.

jakirkham commented:
That said, feel free to merge whichever first.

martindurant commented:
Sorry, I'll require a merger, unfortunately.

jakirkham committed:
Merge remote-tracking branch 'conda-forge/master' into refresh_pinnings

jakirkham commented:
Happy to. Please let me know if that looks ok.
